examples/java/src/main/java/org/apache/beam/examples/WordCount.java

```diff
@@ -48,11 +48,6 @@
  * After you've looked at this example, then see the {@link DebuggingWordCount}
  * pipeline, for introduction of additional concepts.
  *
- * <p>For a detailed walkthrough of this example, see
- *   <a href="https://cloud.google.com/dataflow/java-sdk/wordcount-example">
- *   https://cloud.google.com/dataflow/java-sdk/wordcount-example
```

Comment:
Placeholders exist for this purpose! http://beam.incubator.apache.org/use/walkthroughs/